addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -924,7 +924,7 @@ section, .oe_img_bg, [data-oe-shape-data] {
     // on ".text-gradient *" only to avoid a side effect that makes text nodes
     // wrapped in a span (e.g. bold, italic) no longer have the "gradient
     // crossing the whole text" on them but their own gradient.
-    .o_animated_text, .o_animated_text *, &.o_animated_text * {
+    .o_animated_text, .o_animated_text *, &.o_animated_text *, .fa {
```

Comment:
@rdeodoo Decision is yours:

- Your solution, meaning that we change the behavior in stable just for this damn safari browser
- display: inherit for font-awesome inside text-gradient, meaning we do not change the behavior but we risk breaking anything that relies on icons inside gradients having the default display:inline-block of font-awesome icons (really small risk IMO but...)

I'll merge what you choose, but in any case, I think you should adapt / add a comment :+1:

Reply:
Adapted